src/hb-ot-layout-common.hh

```diff
@@ -1117,8 +1117,9 @@ struct Feature
     ;
 
     out->lookupIndex.serialize (c->serializer, l, it);
-    return_trace (bool (it) || subset_featureParams
-		  || (tag && *tag == HB_TAG ('p', 'r', 'e', 'f')));
```

Comment:
What happens to this `pref` case? There was a reason we added that. Breaks shaping with certain scripts (Khmer?) I think. Probably fonttools doesn't have it?

Reply:
I'll look into it. If we want this then we'll need to check for it during feature collection since if we drop a feature here that the feature collection was planning to keep the remapped feature indices get out of sync with the actual list of subsetted features.

Reply:
> If we want this then we'll need to check for it during feature collection

Right.

Reply:
Oh misread that, this always keeps "pref" which fontTools does too: https://github.com/fonttools/fonttools/blob/main/Lib/fontTools/subset/__init__.py#L1280. I'll check the feature collection code to make sure it's doing the same.

Reply:
I'll fix this in a separate PR, this change won't make the current behaviour any worse since this method now always returns true.